pages/api/upload/index.js

```diff
@@ -0,0 +1,69 @@
+import { v2 as cloudinary } from "cloudinary";
+import formidable from "formidable";
+
+cloudinary.config({
+  cloud_name: process.env.CLOUDINARY_CLOUD_NAME,
+  api_key: process.env.CLOUDINARY_API_KEY,
+  api_secret: process.env.CLOUDINARY_SECRET,
+  secure: true,
+});
+export const config = {
+  api: {
+    bodyParser: false,
+  },
+};
+
+export default async function handler(request, response) {
+  if (request.method === "POST") {
+    const form = formidable({ multiples: true });
+    const [fields, files] = await form.parse(request); //fields nicht löschen, wird für irgendwas gebraucht
```

Comment:
You need fields here, so that you can access the second entry of the array (files)